.circleci/config.yml

```diff
@@ -194,6 +197,11 @@ workflows:
           context: hmpps-common-vars
           requires:
             - request-dev-preview-approval
+      - run_smoke_test:
```

Comment:
Initially added to test circleCI job but later decided to keep it as will be handy to know if smoke tests are failing.